frontend/src/components/theme-toggle-button.tsx

```diff
@@ -23,25 +23,20 @@ function ThemeToggle() {
     }
   }, [isDarkTheme]);
   return (
-    <div>
-      <button
-        name="theme"
-        className={`${
-          isDarkTheme ? 'bg-dark-theme-background' : 'bg-light-theme-background'
-        } flex h-8 w-16 cursor-pointer items-center justify-start rounded-full px-1 py-1`}
-        onClick={toggleTheme}
+    <button
+      name="theme"
+      className={`bg-${isDarkTheme ? 'dark' : 'light'}-theme-background 
+      flex h-8 w-16 cursor-pointer items-center justify-start rounded-full px-1 py-1`}
```

Comment:
Please revert this, the previous one is much readable and we cannot actually use the tailwind classes this way

Reply:
Done Sir